### ISSUE TITLE
Minecraft PVP bot not hunting mobs

### ISSUE THREAD

**Issue by boredcoder411** [PrismarineJS/mineflayer#3344]
## Versions
 - mineflayer: 4.20.0
 - server: vanilla, 1.20.4
 - node: 21.6.2

## Detailed description of a problem
A clear and concise description of what the problem is, with as much context as possible.
What are you building? What problem are you trying to solve?

I followed this youtube video: https://www.youtube.com/watch?v=Giu0ADA5uo8
And I got a bit lazy so I copied the code from the link in the desc: https://raw.githubusercontent.com/TheDudeFromCI/Mineflayer-Youtube-Tutorials/master/pvpBot2/index.js

I already edited the example above to use physicsTick instead of physicTick, which leads me to conclude one of the libraries use are the source of my problems

when I use "guard" in the chat, it responds correctly, and comes to my location. I did just like in the video, I spawned a husk and the bot just stares at it, without responding, and dies.

Any errors?
yes:
(node:19045) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Mineflayer detected that you are using a deprecated event (physicTick)! Please use this event (physicsTick) instead.

however, the code does not crash

## Your current code
see this link: https://raw.githubusercontent.com/TheDudeFromCI/Mineflayer-Youtube-Tutorials/master/pvpBot2/index.js

## Expected behavior
The bot should behave like in this video: https://www.youtube.com/watch?v=Giu0ADA5uo8&t=934s

**Comment by tidalopera** [comment]
Yep. I expect them to guard me well. But they stand there and watch as I am killed by mobs.

**Comment by tidalopera** [comment]
My issue has been resolved.
First, the deprecated physicTick is used at line 51 of mineflayer-pvp/lib/PVP.js.
Second,zombies and spiders are not classified as mobs but as hostiles.
